### ISSUE TITLE
No ranking loss

### ISSUE THREAD

**Issue by FelixZhangC** [dhpollack/stylenet.pytorch#1]
Hello, in the paper the author described two loss, the rank loss and classification loss. In your code, I can't find the related code about rank loss and the classification loss is not cross-entropy mentioned in the paper. I want to know if you did some modification based on the paper? Thank you
